action.yml

```
@@ -308,4 +313,9 @@ runs:
           if [ "${{ runner.os }}" != "Windows" ]; then
             MAYBE_SUDO="sudo -E"
           fi
-          timeout --verbose --kill-after=1s ${TIMEOUT} ${MAYBE_SUDO} tailscale up ${TAGS_ARG} --authkey=${TAILSCALE_AUTHKEY} --hostname=${HOSTNAME} --accept-routes ${ADDITIONAL_ARGS}
+          for i in {1..$RETRY}; do
```

I think this might need to be:
```suggestion
          for ((i=1;i<=$RETRY;i++)); do
```

if we are using an environment variable for it to properly pick up the value here.